### DISCUSSION TITLE
Is there any way to have laravel spark on top of the backpack implementation to manage SaaS subscriptions?

### DISCUSSION THREAD

**Discussion by sanchezpablo** [Laravel-Backpack/community-forum#612] [General] [answered]
Hello, I wonder if Laravel Spark and Backpack can coexist smoothly.

**Comment by karandatwani92** [comment, marked as answer]
Hey @sanchezpablo, Backpack & Spark are two individual packages installable in laravel. Both can be implemented in a laravel project without any conflict.

The way of implementation depends on your project needs, you have to figure out how you want to implement it.
if you wanna show its database records in a CRUD, you can do it via relationship or using laravel spark's models or extending its model to your model.

There is no first-party package from the backpack for laravel spark. You need to figure out the implementation by yourself.
